foxy/distribution.yaml

```diff
@@ -4617,6 +4617,21 @@ repositories:
       url: https://github.com/ijnek/soccer_interfaces.git
       version: rolling
     status: developed
+  sol_vendor:
+    doc:
+      type: git
+      url: https://github.com/OUXT-Polaris/sol_vendor.git
```

Comment:
as this is a vendor package, it's a good idea to also list the upstream license in the `package.xml`, e.g.: https://github.com/ros2/libyaml_vendor/blob/ce1ad8793760ae9585d19a1e484e7342acb95fa2/package.xml#L11.

Could you add that? Thanks!